web/src/engine/predictive-text/worker-thread/src/main/correction/context-tracker.ts

```diff
@@ -444,7 +407,11 @@ export class ContextTracker extends CircularArray<ContextState> {
     // this state is only reached on caret moves; thus, transformDistribution is actually just a single null transform.
     let state = new ContextState(context, model);
     this.enqueue(state);
-    return { state, baseState: null, headTokensRemoved: 0, tailTokensAdded: 0 };
+    const transition = new ContextTransition(state, /* TODO:  we need a clear value here in the future! */ null);
+    // Hacky, but holds the course for now.  This should only really happen from context resets, which can
+    // then use a different path.
+    transition.finalize(state, []);
```

Comment:
Does this need a tracking issue?

Reply:
Addressed in #14442: https://github.com/keymanapp/keyman/pull/14442/files#r2266606447